.travis/publish-images.sh

```diff
@@ -18,8 +18,12 @@ if [ "${DOCKER_PASSWORD}x" != "x" -a "${DOCKER_USERNAME}x" != "x" ]; then
     echo "$DOCKER_PASSWORD" | docker login -u "$DOCKER_USERNAME" --password-stdin
 fi
 
-echo "Building and publishing image ${BUILD_IMAGE}"
-make build docker push BUILD_IMAGE=${BUILD_IMAGE}
+echo "Building image ${BUILD_IMAGE}"
+make build docker
```

Comment:
Doesn't this still need `BUILD_IMAGE...`, for use by the `docker` target?

Reply:
Good catch!